patterns/just-arrived-full-hero.php

```diff
@@ -0,0 +1,23 @@
+<?php
+/**
+ * Title: WooCommerce Just Arrived Full Hero
+ * Slug: woocommerce-blocks/just-arrived-full-hero
+ * Categories: WooCommerce
+ */
+?>
+<!-- wp:cover {"useFeaturedImage":true,"dimRatio":0,"minHeight":50,"minHeightUnit":"vw","contentPosition":"center center","isDark":false,"align":"full","style":{"spacing":{"padding":{"left":"50vw"}}}} -->
+<div class="wp-block-cover alignfull is-light" style="padding-left:50vw;min-height:50vw"><span aria-hidden="true" class="wp-block-cover__background has-background-dim-0 has-background-dim"></span><div class="wp-block-cover__inner-container"><!-- wp:heading {"style":{"spacing":{"margin":{"top":"0px","right":"0px","bottom":"0px","left":"0px"}},"typography":{"fontSize":"40px","fontStyle":"normal","fontWeight":"700"}}} -->
+<h2 id="just-arrived" style="margin-top:0px;margin-right:0px;margin-bottom:0px;margin-left:0px;font-size:40px;font-style:normal;font-weight:700">Just arrived</h2>
+<!-- /wp:heading -->
+
+<!-- wp:paragraph {"style":{"typography":{"fontSize":"16px"}}} -->
```

Comment:
This is another place we use a fixed value. I think a preset value is more versatile for a pattern.